README.md

```diff
@@ -8,4 +8,8 @@ Today we are learning the basics of Git & GitHub, how it works, etc. as a part o
 
 ### Hosts
 
+* PRERANA PRABHU
+* PRAVEEN KUMR PURSHOTHAM
```

Comment:
First of all, thanks for committing to my repository. You'll be getting a green card on your profile. Couple of changes.

1. If you use all caps online, it's like shouting. So please refrain from using all caps. Convert it to sentence case.
2. My name is completely murdered. It's supposed to be `Praveen Kumar Purushothaman`.

```suggestion
* Prerana Prabhu
* Praveen Kumar Purushothaman
```

Finally, please remove all the unwanted spaces at the end.